source/upgrade.txt

```diff
@@ -61,6 +61,12 @@ Version 4.8 Breaking Changes
   earlier. To learn more about this change, see the :ref:`<java-server-release-change-v4.8>`
   section.
 
+- The ``RecordCodec``, implemented in v4.6, deserialized POJOs and record
+  classes that were specified as type parameters in ``List`` or ``Map`` fields
+  of a record as ``Document`` values instead of their respective classes. This
```

Comment:
The sentence refers to two different things. The first mention refers to "POJOs and record classes that were specified as type parameters" and the second is qualifying that they are List or Map fields within a record.

I'll add an example just in case it isn't clear to all readers.